src/Common/tests/System/Net/Configuration.Certificates.cs

```diff
@@ -29,8 +31,15 @@ public static partial class Certificates
 
             private static X509Certificate2Collection GetCertificateCollection(string certificateFileName)
             {
+                // On Windows, .Net Core applications should not import PFX files in parallel to avoid a known system-level race condition.
```

Comment:
Is this specific to .NET Core?  From the discussion, since it's an OS-level issue, I'd have expected this problem to exist on desktop as well, no?